frameit/lib/frameit/strings_parser.rb

```diff
@@ -10,7 +10,7 @@ def self.parse(path)
       result = {}
 
       # A .strings file is UTF-16 encoded. We only want to deal with UTF-8
-      content = `iconv -f UTF-16 -t UTF-8 '#{path}' 2>&1`
+      content = `iconv -f UTF-16 -t UTF-8 "#{path}" 2>&1`
```

Comment:
Wouldn't it be better to use a cross platform way of escaping things?

This is how we solved it in u3d: https://github.com/DragonBox/u3d/blob/master/lib/u3d_core/core_ext/string.rb#L27-L33

Reply:
Is this an escaping problem?

I thought it was just an `iconv` implementation detail that required the path to be wrapped in `"` on Windows instead of `"` _or_ `'` on Linux or macOS. Did I get that wrong?

Reply:
Maybe. Not sure!

Reply:
I just think that it is the way Windows requires its parameters to be passed. Single quotes have no meaning there. Not specific to `iconv` in my mind.

Reply:
Ok. What would be the best solution here then?

Reply:
I think a cross platform way of quoting things would be ideal on the long run for running generic commands. The fact that double quote works on both is probably acceptable here given that the command is explicit and not built at runtime. Maybe comment that double quotes required to work on LMW.

Reply:
added a comment